gtk4/src/auto/entry_buffer.rs

```diff
@@ -64,16 +64,6 @@ pub trait EntryBufferExt: 'static {
     #[doc(alias = "gtk_entry_buffer_emit_inserted_text")]
     fn emit_inserted_text(&self, position: u32, chars: &str, n_chars: u32);
 
-    fn get_property_length(&self) -> u32;
```

Comment:
Why do these all disappear?

Reply:
Because there are manual version of them, somehow marking them as ignore didn't actually ignore them?

Reply:
That seems like a bug in gir :)